### PR TITLE
feat: add polygon flis to legacy

### PR DESCRIPTION
**Pull request by janndriessen**
## **Summary of Changes**

* Adds deprecated and delevered polygon tokens to legacy

## **Test Data or Screenshots**
Bildschirmfoto 2024-12-10 um 15 46 17

###### _By submitting this pull request, you are confirming the following to be true:_

- I have reviewed the [Contribution Guidelines](https://github.com/IndexCoop/index-app/blob/master/CONTRIBUTING.md).
- I have performed a self-review of my own code.
- I have updated my repository to match the master at `IndexCoop/index-app`.
- I have included test data or screenshots that prove my fix is effective or that my feature works.

### CODE REVIEW

File: src/app/legacy/components/redeem-widget/use-formatted-data.tsx [hunk 1]
```diff
@@ -42,7 +42,7 @@ export function useFormattedData() {
   if (legacyQuote) {
     const { outputTokens, outputTokenPricesUsd, units } = legacyQuote
     outputAmounts = outputTokens.map((outputToken, index) =>
-      formatAmount(Number(formatWei(units[index], outputToken.decimals))),
+      formatAmount(Number(formatWei(units[index], outputToken.decimals)), 4),
```
Comment (janndriessen) [inline src/app/legacy/components/redeem-widget/use-formatted-data.tsx:45]: better visibility for components like WBTC (which usually have small amounts)

File: src/app/legacy/polygon.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import { Token } from '@/constants/tokens'
```
Comment (tokdaniel) [inline src/app/legacy/polygon.ts:1]: why don't just use the tokenlists here? Or have you just released and thats why?

**Review by tokdaniel — APPROVED** [review]
lgtm and all, but probably we should update with the tokenlist stuff, and move most of these there. Idk though

**Review by 0xonramp — APPROVED** [review]
🔥

File: src/app/legacy/polygon.ts (hunk 2 above)
Comment (janndriessen) [reply src/app/legacy/polygon.ts:1]: yea I was lazy 😅 started out with this file but then needed the components. should have added the indices there too. will update tokenlists.

**Review by tokdaniel — APPROVED** [review]

**Pull request merged**